### PR TITLE
Fix maven badges in READMEs

### PR DESCRIPTION
**Pull request by mziccard**

### CODE REVIEW

File: google-cloud-contrib/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Packages that provide higher-level abstraction/functionality for common Google C
 
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/google-cloud-java.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/google-cloud-java)
 [![Coverage Status](https://coveralls.io/repos/GoogleCloudPlatform/google-cloud-java/badge.svg?branch=master)](https://coveralls.io/r/GoogleCloudPlatform/google-cloud-java?branch=master)
-[![Maven](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-bigquery.svg)]( https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-bigquery.svg)
+[![Maven](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-contrib.svg)]( https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-contrib.svg)
```
Comment (lesv) [inline google-cloud-contrib/README.md]: What's this?

**Comment by lesv** [comment]
LGTM - with one Q.

File: google-cloud-contrib/README.md (hunk 1 above)
Comment (mziccard) [inline google-cloud-contrib/README.md]: What do you mean with this? The contrib artifact? Contains contributions for common use cases that are not part of the `google-cloud` artifact but usually build on top of it (README [here](https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-contrib). An example is the NIO gcs filesystem provided.

Comment (lesv) [inline google-cloud-contrib/README.md]: Got it -- meant to go look, but trying to deal w/ way too much stuff today.

**Pull request merged**